modules/cert-manager-override-environment-variables.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-override-environment-variables_{context}"]
+= Customizing cert-manager by overriding environment variables from the cert-manager Operator API
+
+You can override the default environment variables for the {cert-manager-operator} by adding a `spec.controllerConfig` section in the `CertManager` resource.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Edit the `CertManager` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit certmanager cluster
+----
+
+. Add a `spec.controllerConfig` section with the following override arguments:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  name: cluster
+  ...
+spec:
+  ...
+  controllerConfig: <1>
+    overrideEnv:
+      - 'HTTP_PROXY'
+      - 'HTTPS_PROXY'
+      - 'NO_PROXY'
+----
+<1> Add the `spec.controllerConfig` section.
+
+. Save the file to apply the changes.
+
+.Verification
+
+* Verify that the cert-manager pods are running by entering the following command:
++
+[source,terminal]
+----
+$ oc get pods -n cert-manager -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+name: cert-manager-6d48f88495-c88gd
+namespace: cert-manager
+...
+env:
+      - name: POD_NAMESPACE
+        valueFrom:
+          fieldRef:
+            apiVersion: v1
+            fieldPath: metadata.namespace
+      - name: HTTP_PROXY
```

Comment:
The indentations are not correct. And `POD_NAMESPACE` needs not be documented, because unrelated.
```suggestion
$ oc get pod <redeployed_cert-manager_pod> -n cert-manager -o yaml
----
+
.Example output. The overridden environment variables are added under the `env` field.
[source,yaml]
----
    - name: HTTP_PROXY
      value: http://<PROXY_URL>
    - name: HTTPS_PROXY
      value: https://<PROXY_URL>
    - name: NO_PROXY
      value: <IGNORE_PROXY_DOMAINS>
```